tests/slow_growth_test.py

```diff
@@ -0,0 +1,27 @@
+import pytest
```

Comment:
Please add these tests to the existing file where the other tests for `SmolyakGridGenerator` are, rather than a separate file.

Reply:
Done.